mlperf_logging/result_summarizer/result_summarizer.py

```diff
@@ -106,17 +108,27 @@ def _code_url(system_desc, ruleset):
 
 
 def _row_key(system_desc):
-    system_name = system_desc['system_name']
+    system_name = '{}-{}'.format(system_desc['system_name'], system_desc['framework'])
```

Comment:
@bitfort @xyhuang This change is not related to the olympic score rule.

In the future, can we add such changes in a separate PRs? It is easy to miss such small, unrelated changes.